egs/swbd/s5c/local/run_durmodel.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+[ -f ./path.sh ] && . ./path.sh; # source the path.
+. cmd.sh
+. parse_options.sh || exit 1;
+
+system=exp/tri4
```

Comment:
we often use srcdir for a directory where we get a model from, or alidir
from a directory where we also need the alignments to be present.

On Sat, Jan 23, 2016 at 8:26 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/swbd/s5c/local/run_durmodel.sh
> <https://github.com/kaldi-asr/kaldi/pull/414#discussion_r50628686>:
>
> > @@ -0,0 +1,31 @@
> > +#!/bin/bash
> > +
> > +[ -f ./path.sh ] && . ./path.sh; # source the path.
> > +. cmd.sh
> > +. parse_options.sh || exit 1;
> > +
> > +system=exp/tri4
>
> system is a vague name, use a more descriptive name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/414/files#r50628686>.
>